components/molecules/tabs-filter.tsx

```diff
@@ -0,0 +1,48 @@
+import {ArrowLeftIcon, ArrowRightIcon} from '@heroicons/react/outline'
+import {useRef} from 'react'
+import {useRouter} from 'next/router'
+import Button from '../atoms/button'
+import Link from 'next/link'
+import classNames from '../../utils/classes'
+
+export type TabsFilterProps = {
+  tags: Array<string>
+}
+
+export default function TabsFilter({tags}: TabsFilterProps) {
+  const slider = useRef(null)
+  const router = useRouter()
+
+  function slideToRight() {
+    slider.current.scrollBy({left: 200, behavior: 'smooth'})
```

Comment:
You can do this with only one function named `slideTo(direction: 'right'|'left')`